x/auth/spec/README.md

```diff
@@ -21,24 +21,19 @@ This module will be used in the Cosmos Hub.
 ## Contents
 
 1. **[Concepts](01_concepts.md)**
-    - [Gas & Fees](01_concepts.md#gas-&-fees)
+   - [Gas & Fees](01_concepts.md#gas-&-fees)
 2. **[State](02_state.md)**
-    - [Accounts](02_state.md#accounts)
+   - [Accounts](02_state.md#accounts)
 3. **[Messages](03_messages.md)**
```

Comment:
Since `x/auth` doesn't actually have messages, would it be more clear to have this file just called `AnteHandlers` with the contents of the file together wit ha bit more of an overview of the function of the ante handler and how it works?

Seems strange for that to live within the messages file. Even though remove `messages.md` would not adhere to SPEC-SPEC, i think it would be a positive change, given how auth is not a standard module

Reply:
Alternatively instead of removing hte file, we could just have a messages file that simply says "the x/auth module does not implement any messages" and move AnteHandler documentation into its own file.

Reply:
Makes sense. I renamed to `antehandler.md`. Anyways, there's another `vesting.md`, so x/auth really doesn't adhere to SPEC-SPEC.